core/qt-ble.h

```diff
@@ -28,6 +28,7 @@ class BLEObject : public QObject
 	dc_status_t write(const void* data, size_t size, size_t *actual);
 	dc_status_t read(void* data, size_t size, size_t *actual);
 	dc_status_t get_name(char *res, size_t size);
+	dc_status_t read_characteristic(const QBluetoothUuid &uuid, char *res, size_t size);
```

Comment:
This should be private - again this will make it easier to read the code.

Reply:
It can't be private, because it's used by `qt_ble_ioctl()`